arbitrum-docs/node-running/how-tos/running-a-daserver.mdx

```diff
@@ -0,0 +1,536 @@
+---
```

Comment:
@TucksonDev it looks like commit f8586d86a77d331149bc3f5ae050e6480db2dbe5 removed this; need to take a closer look at the DAC docs you recently introduced, to determine the relationship between this and those...

Reply:
The new pages replace this one, so this should be removed. Did you see anything that's not in the new page?
(specially [this one](https://arbitrum-docs-git-ia-35-offchain-labs.vercel.app/node-running/how-tos/data-availability-committee/deploy-a-das))? 🙏